.github/mergify.yml

```diff
@@ -12,10 +14,26 @@ pull_request_rules:
         type: "APPROVE"
         message: "We've automatically approved this PR because the checks from the automated Pull Request have passed."
 
+  - name: "approve manual PRs that have passed checks"
+    conditions:
+      - "-check-failure~=.*alpine.*"
+      - "-check-failure~=.*package.*"
+      - "-check-failure~=.*matrix.*"
+      - "check-success~=.*build-and-push.*"
+      - "#files<=14"
```

Comment:
I think we said it should approve only when the files are in `vendor/*`

With this logic, it will approve even PRs by a `CODEOWNER` to `tasks/`, which should still have an approval.